msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -101,22 +101,22 @@ A meeting lifecycle includes pre-meeting, in-meeting, and post-meeting app exper
 
 ## User types in Teams
 
-Teams supports user types, such as in-tenant, guest, federated or external user in a Teams meeting. Each user type can have one of the [user roles in Teams meeting](#user-roles-in-teams-meeting).
+Teams supports user types, such as in-tenant, guest, federated or external, and anonymous users in a Teams meeting. Each user type can have one of the [user roles in Teams meeting](#user-roles-in-teams-meeting).
 
 > [!NOTE]
 >
-> Currently when a third person is added to a one-on-one call, the call is elevated to a group call that means a new session starts. Apps added to the one-on-one call are not available in the group call. However, they can be added again.
+> Currently, when a third person is added to a one-on-one call, the call is elevated to a group call that means a new session starts. Apps added to the one-on-one call aren't available in the group call. However, they can be added again.
```

Comment:
[Grammar] The sentence below is a run-on sentence. It needs a conjunction between the two main clauses. Also "currently" suggests that we plan to change this, so I'd rather we omitted it.
> Currently, when a third person is added to a one-on-one call, the call is elevated to a group call that means a new session starts.

Consider: "When a third person is added to a one-on-one call, the call is elevated to a group call, which starts a new session."

Reply:
Considered and edited the text as "When a third person is added to a one-on-one call, the call is elevated to a group call, which starts a new session."